scripts/layouts_tool/layoutsfilehandler.h

```diff
@@ -0,0 +1,73 @@
+#ifndef LAYOUTSFILEHANDLER_H
+#define LAYOUTSFILEHANDLER_H
+
+#include <QFile>
+#include <QDir>
+#include "layout.h"
+
+typedef QList<QStringList> LayoutNamesData;
+typedef KeyboardLayoutPointer (*GetLayout_t)(std::string layoutName);
+
+class LayoutsFileHandler {
+public:
+    LayoutsFileHandler();
+    virtual ~LayoutsFileHandler();
+
+    void open(QString &layoutsPath, QList<Layout> &layouts);
+    void save(QFile &f, QList<Layout> &layouts);
+
+    static const QString INDENT;
```

Comment:
Done, I placed `INDENT` and all other static const QString and QStringList objects in an anonymous namespace in the cpp file. Also renamed them all to camelcase prefixed with `k`.